robot/impl/robot_reconfigure_test.go

```diff
@@ -2372,6 +2372,7 @@ func TestStatusServiceUpdate(t *testing.T) {
 }
 
 func TestRemoteRobotsGold(t *testing.T) {
+	t.Skip()
```

Comment:
is there a JIRA to not skip these or delete?

Reply:
RSDK-688 to re enable them when we figure out what's the issue with the CI